integration_test/support/migration.exs

```diff
@@ -67,19 +67,13 @@ defmodule Ecto.Integration.Migration do
     false = exists? table(:users)
 
     create table(:users) do
-      add :name, :string
-      add :to_be_removed, :string
-    end
-
-    true = exists? table(:users)
-
-    alter table(:users) do
-      modify :name, :text
+      add :name, :text
       add :custom_id, :uuid
-      remove :to_be_removed
       timestamps
     end
 
+    true = exists? table(:users)
```

Comment:
Sounds good.  In that case do you have a good name for the new test file?  `alter_table.exs` doesn't fit as well if we'll be testing the creating and dropping of tables and indices, too.  Maybe, `table_index.exs`?